pages/settings/activity.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <v-container>
+    <v-row>
+      <v-col cols="12" :offset-md="1" md="5" class="pt-0 pb-4">
+        <!-- Administrator settings -->
+        <v-list
+          v-if="$auth.user.Policy.IsAdministrator"
```

Comment:
This should probably be handled via a [router middleware](https://nuxtjs.org/examples/middleware-router) instead. It'd check this same policy, and redirect to the home page if the user is trying to access an admin page without being an admin.

Reply:
middleware created :)